code/modules/lighting/lighting_turf.dm

```diff
@@ -48,7 +46,7 @@
 	var/ambient_g = 0
 	var/ambient_b = 0
 
-	if (ambient_light)
+	if (ambient_light && ambient_light_multiplier) // If either of these are false-y we can use the simplier path and avoid calculations
```

Comment:
It's technically undefined behavior for ALM to be zero, but I don't have a problem with this change otherwise.